easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2018.01.01-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PackedBinary"
+
+name = "EvidentialGene"
+version = '2018.01.01'
+
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://arthropods.eugenes.org/EvidentialGene/'
+description = """EvidentialGene is a genome informatics project for 
+ "Evidence Directed Gene Construction for Eukaryotes",
+ for constructing high quality, accurate gene sets for
+ animals and plants (any eukaryotes), being developed by
+ Don Gilbert at Indiana University, gilbertd at indiana edu."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['ftp://arthropods.eugenes.org/evigene_old/']
+month = ['', 'jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
+dlver = version.split('.')[0][-2:]
+dlver += month[int(version.split('.')[1])]
+dlver += version.split('.')[2]
+sources = ['evigene%s.tar' % dlver]
```

Comment:
Ugh, this is a bit messy, creative versioning, yaay!

I would propose to clean this up a little bit:

```python
month = ['jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
dlver = '%(version_major)s'[-2:] + month[int('%(version_minor)s') - 1] + version.split('.')[2]

Reply:
done